### ISSUE TITLE
EF Power Tools link?

### ISSUE THREAD

**Issue by KirkMunro** [aspnet/Docs#7724]
You mention EF Power Tools used to generate the entity framework diagrams, but you don't provide a link. This is in reference to this sentence:

&gt; This diagram was generated using EF Power Tools for EF 6.x. Creating the diagram isn't part of the tutorial.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 634dfb2c-2626-9ec5-35fe-2bd2f27c5e23
* Version Independent ID: 7a77b058-f005-8d68-ae46-e487c597f18e
* Content: [Razor Pages with EF Core in ASP.NET Core - Data Model - 5 of 8](https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/complex-data-model?view=aspnetcore-2.1&tabs=netcore-cli)
* Content Source: [aspnetcore/data/ef-rp/complex-data-model.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/data/ef-rp/complex-data-model.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Toby9876** [comment]
I installed this: https://marketplace.visualstudio.com/items?itemName=ErikEJ.EntityFramework6PowerToolsCommunityEdition 
and it did not work. When trying to run "View Entity Data Model" on SchoolContext I get the error message "A constructible type deriving from DbContext could not be found in the selected file". As far as I can tell, this is not true. I tried adding an extra constructor with no paramters, but this did not help either.
